SPECS/kernel-azure/config

```diff
@@ -7243,6 +7244,7 @@ CONFIG_DEBUG_INFO=y
 # CONFIG_DEBUG_INFO_SPLIT is not set
 CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 # CONFIG_DEBUG_INFO_DWARF4 is not set
+# CONFIG_DEBUG_INFO_DWARF5 is not set
```

Comment:
Result of this change https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/0c168d7f36d5c63c9568f5cf2a3910822499a742